Composer/packages/form-dialogs/src/components/common/CommandBarUpload.tsx

```diff
@@ -27,6 +27,7 @@ export const CommandBarUploadButton = (props: Props) => {
   const onChange = () => {
     if (inputFileRef.current.files) {
```

Comment:
Should this be current?.files  - current for a ref can be null right (although for a UI callback very unlikely)

Reply:
If the ref is not set that's a major bug in react.